web/application.py

```diff
@@ -701,13 +701,12 @@ def unloadhook(h):
     def processor(handler):
         try:
             result = handler()
-            is_gen = is_iter(result)
         except:
             # run the hook even when handler raises some exception
             h()
             raise
 
-        if is_gen:
+        if result and hasattr(result, "__next"):
```

Comment:
Typo here. It should be `__next__`. 

Also, why is cleanup of `is_iter` part of MySQLdb PR?

Reply:
Fixed the `__next` typo.